src/tellor_disputables/cli.py

```diff
@@ -184,15 +185,17 @@ async def start(
                     del display_rows[0]
 
                 # Display table
-                _, times, links, _, values, disputable_strs, assets, currencies = zip(*display_rows)
+                _, times, links, query_type, values, disputable_strs, assets, currencies, chain = zip(*display_rows)
+
                 dataframe_state = dict(
                     When=times,
                     Transaction=links,
-                    # QueryType=query_types,
+                    QueryType=query_type,
                     Asset=assets,
                     Currency=currencies,
-                    Value=values,
+                    Value=[f"{str(val)[:6]}...{str(val)[-5:]}" if len(str(val)) > 10 else val for val in values],
```

Comment:
What's this one-liner doing? Might want to add some explanation as a comment for future readers.